packages/strapi-plugin-content-manager/admin/src/components/DisplayedFieldsDropdown/ItemDropdown.js

```diff
@@ -0,0 +1,25 @@
+import { DropdownItem } from 'reactstrap';
+import styled from 'styled-components';
+
+const ItemDropdown = styled(DropdownItem)`
+  padding: 0;
+  padding-top: 9px;
+  &:active,
+  :focus {
```

Comment:
Did you mean `&:focus` selector ? 
ItemDropdown's focus state won't be the one above without &: selector